<launch>
  <arg name="base" value="kobuki"/>  <!-- used to be base -->
  <arg name="stacks" value="hexagons"/>  <!-- used to be stacks -->
  <arg name="3d_sensor" value="hokuyo"/> <!-- used to be 3d_sensor -->


  <arg name="urdf_file" default="$(find xacro)/xacro.py 
	'$(find turtlebot_description)/robots/$(arg base)_$(arg stacks)_$(arg 3d_sensor).urdf.xacro'" />
  <param name="robot_description" command="$(arg urdf_file)" />

  <group ns="robot1">
    <param name="tf_prefix" value="robot1_tf" />
    <include file="$(find my_2d_nav)/launch/single_robot.launch" >
      <arg name="init_pose" value="-x -1 -y -1 -z 0" />
      <arg name="robot_name"  value="Robot1" />
    </include>
	</group>

  <group ns="robot2">
    <param name="tf_prefix" value="robot2_tf" />
    <include file="$(find my_2d_nav)/launch/single_robot.launch" >
      <arg name="init_pose" value="-x -3 -y -1 -z 0 -Y 3.14" />
      <arg name="robot_name"  value="Robot2" />
    </include>
	</group>
</launch>
